Locker/Locker/Locker.swift

```diff
@@ -0,0 +1,246 @@
+//
+//  Locker.swift
+//  Locker
+//
+//  Created by Zvonimir Medak on 19.10.2021..
+//  Copyright © 2021 Infinum. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+@objcMembers
+public class Locker: NSObject {
+
+    // MARK: - Public properties
+
+    public static var userDefaults: UserDefaults? {
+        get {
+            return currentUserDefaults == nil ? UserDefaults.standard : currentUserDefaults
+        }
+        set {
+            currentUserDefaults = currentUserDefaults == nil ? UserDefaults.standard : newValue
```

Comment:
```suggestion
            currentUserDefaults = newValue ?? .standard
```
Shouldn't we check if `newValue` is nil, and if so, use standard UserDefaults? 🤔

Reply:
That makes much more sense 😄